0099-Replacing bash as the default shell.md

```diff
@@ -0,0 +1,121 @@
+---
+feature: oil-shell
+start-date: 2021-08-11
+author: Raphael Megzari (happysalada)
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues:
+---
+
+# Summary
+
+[summary]: #summary
+
+Use oilshell as the default shell for nixos. Oilshell aims to be compatible with bash while trying to bring real programming language features (like hashmap or dicts) to the shell.
```

Comment:
Styling issue: Is it Oilshell or oilshell?

Reply:
"Use oilshell as the default shell for nixos" seems to indicate that you would like to change the value of this option `users.defaultUserShell`. From the rest of the RFC, I get that the proposal is rather to change the default change in `stdenv`. This is not the same and both changes would be pretty much independent, so please clarify this point.

Reply:
I am only suggesting it for stdenv for now actually. Let me correct this.

Reply:
Feel free to comment again if you think this is still ambiguous